### ISSUE TITLE
Integrating XCStrings Tool into a Swift Package Target has wrong package git url

### ISSUE THREAD

**Issue by gewill** [liamnichols/xcstrings-tool#41]
[Integrating XCStrings Tool into a Swift Package Target](https://swiftpackageindex.com/liamnichols/xcstrings-tool/documentation/documentation/integrating-xcstrings-tool-into-a-swift-package-target) has wrong package git url:

```swift
        // 1. Add the xcstrings-tool Package dependency
        .package(url: "https://github.com/liamnichols/xcstrings-tool-package.git", from: "0.1.0")
```

The correct package git url should be:
```swift
        // 1. Add the xcstrings-tool Package dependency
        .package(url: "https://github.com/liamnichols/xcstrings-tool.git", from: "0.1.0")
```

**Comment by liamnichols** [comment]
Hi @gewill, thanks for raising this. It was fixed in #33 however I haven't released a new version yet so it isn't reflected in the docs unless you visit the version for `main`:

https://swiftpackageindex.com/liamnichols/xcstrings-tool/main/documentation/documentation/integrating-xcstrings-tool-into-a-swift-package-target

Please note that the best URL is actually `https://github.com/liamnichols/xcstrings-tool-plugin.git` if you want to avoid having to compile SwiftSyntax on every clean build 🙇 

I'll try to release a new version this week that makes this clearer in the docs 👍

**Comment by gewill** [comment]
Thanks for the tips and this great package you developed👍
